Firmware/MotorControl/controller.hpp

```diff
@@ -96,7 +97,8 @@ class Controller : public ODriveIntf::ControllerIntf {
     bool anticogging_valid_ = false;
 
     // custom setters
-    void set_input_pos(float value) { input_pos_ = value; input_pos_updated(); }
+    void set_input_pos(float value) { input_pos_ = value; input_pos_updated();}
```

Comment:
formatting: `input_pos_updated();}` -> `input_pos_updated(); }`

Reply:
Fixed in a262688